acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/signer/SignerConfiguration.java

```diff
@@ -20,12 +20,11 @@
   private static final int TCP_PORT = 8845;
   private static final Duration POLLING_INTERVAL = Duration.ofMillis(500);
 
-  /** ChainId defined in the Pantheon dev mode genesis. */
-  private static final String CHAIN_ID = "2018";
-
+  private final String chainId;
   private final String hostname;
 
-  public SignerConfiguration(final String hostname) {
+  public SignerConfiguration(final String chainId, final String hostname) {
```

Comment:
is Port not configurable?

Reply:
Not yet, that comes later (proper multithreading of the AT)